### PR TITLE
Update office-365-u.s.-government-gcc-high-endpoints.md

### LINKED ISSUE #12098: Include caveat for Cross Cloud Anonymous Join for the Teams endpoint section
Teams in GCCH supports Cross-Cloud Anonymous Join, which means a GCCH Teams user/client can join DoD or WW hosted meetings. This has caused some confusion/lead to customers raising cases because they have whitelisted for GCCH, but not for the other clouds. It is currently only implied that if they want to be able to connect to WW/DoD meetings, that they have to whitelist for those endpoints.

Could we please include a warning/notification under the Teams subsection that for Cross-Cloud Join to properly work, they must whitelist the endpoints for the cloud of the target meeting? Also add links to the other cloud endpoint pages.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9a9f2948-6ae1-f907-836b-07f108ddf828
* Version Independent ID: 9a9f2948-6ae1-f907-836b-07f108ddf828
* Content: [Office 365 U.S. Government GCC High endpoints - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note for Cross-Cloud Join

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12098

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/includes/office-365-u.s.-government-gcc-high-endpoints.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
 31 | Allow<BR>Required | Yes | `*.gov.skypeforbusiness.us, *.gov.teams.microsoft.us, gov.teams.microsoft.us`<BR>`52.127.88.0/21, 104.212.44.0/22, 195.134.228.0/22` | **TCP:**  443, 80
-
-## Microsoft 365 Common and Office Online
+  
+Notes for this table:
+
+For  Cross-Cloud Join to properly work, you must whitelist the endpoints for the cloud of the target meeting
+  
+  - [Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
+  - [Office 365 U.S. Government DoD endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints)
```
Comment (JohanFreelancer9) [inline microsoft-365/includes/office-365-u.s.-government-gcc-high-endpoints.md]:
```suggestion
> [!NOTE]
> For  Cross-Cloud Anonymous Join to work properly, you must add these endpoints for the cloud of the target meeting to the safe senders list:
> - [Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
> - [Office 365 U.S. Government DoD endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints)
```

File: microsoft-365/includes/office-365-u.s.-government-gcc-high-endpoints.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
+
```
Comment (JohanFreelancer9) [inline microsoft-365/includes/office-365-u.s.-government-gcc-high-endpoints.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**